src/api/routes/rosetta/account.ts

```diff
@@ -122,22 +121,21 @@ export function createRosettaAccountRouter(db: PgStore, chainId: ChainID): expre
           ];
 
           // Add Fungible Token balances.
-          if (isFtMetadataEnabled()) {
```

Comment:
Is there still a way for Rosetta users to disable ft metadata? I think at least 1 entity isn't using it, and idk if they might run into problems if it's forced enabled for them.

Reply:
Hmm yeah good point, I'll re-add this ENV